guides/common/modules/proc_using-container-registries.adoc

```diff
@@ -30,3 +33,39 @@ Pulling container images:
 ----
 # podman pull {foreman-example-com}/my-image:<optional_tag>
 ----
+
+Pushing container images to the {Project} container registry:
+
+* To indicate which organization, product, and repository the container image belongs to, use container namespaces.
+* All labels are composed of entirely lowercase characters regardless of actual label capitalization.
+* Pushes follow one of two formats:
+[options="nowrap", subs="+quotes,attributes"]
+----
+# podman push <image> example.com/<organization_label>/<product_label>/<repository_name>[:tag]
+# podman push <image> example.com/id/<organization_id>/<product_id>/<repository_name>[:tag]
+----
+
+* A complete push looks like the following:
```

Comment:
```suggestion
* An example of a complete push:
```
The meaning is basically the same, but using the keyword 'example' would really help users scan the text more quickly.